man/check_journal.1.ronn

```diff
@@ -44,6 +44,11 @@ systems.
 * `-j`, `--journalctl` <PATH>:
     Specifies the journalctl executable.
 
+* `-L`, `--no-limit`:
```

Comment:
This seems to be a hickup, could you double check why the -L option is documented twice now?

Reply:
Guess a copy & paste fail? I have no idea how that happened, but its fixed.